api/network.go

```diff
@@ -0,0 +1,11 @@
+package api
+
+// IsIngressNetwork check if the network is an ingress network
+func IsIngressNetwork(nw *Network) bool {
```

Comment:
I also think ``github.com/docker/swarmkit/manager/network` would be a fine place to put these. In particular, `IsIngressNetworkNeeded` is basically business logic, so the `manager/network` package might make more sense.